templates/streaming-media-master.yml

```diff
@@ -115,42 +109,37 @@ Resources:
       - ArtifactBucket
       - IAMRoleStack
     Properties:
-      TemplateURL:
-        !Sub
-          - 'https://${S3Bucket}.s3.${AWS::URLSuffix}/${S3KeyPrefix}/templates/ci-pipeline.yml'
-          - S3Bucket: !If [UsingDefaultBucket, !Sub '${S3BucketName}-${AWS::Region}', !Ref S3BucketName]
+      TemplateURL: !Sub 'https://${S3BucketName}.s3.${AWS::URLSuffix}/${S3KeyPrefix}/templates/ci-pipeline.yml'
       Parameters:
         GitHubUser: !Ref GitHubUser
         GitHubRepoName: !Ref GitHubRepoName
         SourceRepoBranch: !Ref SourceRepoBranch
         ReleaseBranch: !Ref ReleaseBranch
         GitHubOAuthToken: !Ref GitHubOAuthToken
         ArtifactBucket: !Ref ArtifactBucket
-        # GitMergeLambda: !Ref GitMergeLambda
         CodePipelineRoleArn: !GetAtt
           - IAMRoleStack
           - Outputs.CodePipelineRoleArn
         CodeBuildRoleArn: !GetAtt
           - IAMRoleStack
           - Outputs.CodeBuildRoleArn
+        GitMergeRoleArn: !GetAtt
+          - IAMRoleStack
+          - Outputs.GitMergeRoleArn
   EmptyBucketLambdaStack:
     Type: 'AWS::CloudFormation::Stack'
     DependsOn:
       - ArtifactBucket
     Properties:
-      TemplateURL:
-        !Sub
-          - 'https://${S3Bucket}.s3.${AWS::URLSuffix}/${S3KeyPrefix}/templates/empty-bucket.yml'
-          - S3Bucket: !If [UsingDefaultBucket, !Sub '${S3BucketName}-${AWS::Region}', !Ref S3BucketName]
+      TemplateURL: !Sub 'https://${S3BucketName}.s3.${AWS::URLSuffix}/${S3KeyPrefix}/templates/empty-bucket.yml'
       Parameters:
         BucketName: !Ref ArtifactBucket
   MediaStorageStack:
     Type: 'AWS::CloudFormation::Stack'
+    DependsOn:
+      - ArtifactBucket
     Properties:
-      TemplateURL:
-        !Sub
-          - 'https://${S3Bucket}.s3.${AWS::URLSuffix}/${S3KeyPrefix}/templates/media-storage.yml'
```

Comment:
`${S3BucketName}-${AWS::Region}` bucket cannot be recognized during cloud formation deploy,
alter to `$S3BucketName` to allow bucket reference